examples/inheritance/controls/example.rb

```diff
@@ -9,3 +9,6 @@
     impact 0.0
   end
 end
+
```

Comment:
The example is great, I think we should not expose that to our customer examples yet. Could we move those to `test/unit/mock/profiles`?